app/assets.js

```diff
@@ -39,7 +41,28 @@ function concat_js(files) {
   log.info(
     `compiling ${relative(config.js.sourceFiles)} -> ${relative(outFile)}`);
 
-  concatfiles(files, outFile);
+  concatfiles(files, outFile, function(err) {
+    if(err) {
+      throw err;
+    }
+
+    transpile_js();
+  });
+}
+
+
+function transpile_js() {
+  let inFile = path.join(config.js.outDir, config.js.filename);
+  let outFile = path.join(config.js.outDir, 'transpiled_' + config.js.filename);
+
+  log.info(
+    `transpiling ${relative(inFile)} -> ${relative(outFile)}`);
+
+  babel.transformFile(inFile, { "presets": ["es2015"] }, function(err, result) {
```

Comment:
Would it make sense to get this `{"presets": ["es2015"]}` from `config.js`?

Reply:
Yes, good call. Will do that.